CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 All notable changes to this project will be documented in this file.
 
-## Release 0.5.0
+## Release 0.5.1
```

Comment:
I think the release version should be either `0.4.1` or `0.5.0` since there's currently no `0.5.0` release.